src/include/cell_inline.h

```diff
@@ -1429,34 +1429,31 @@ __wt_cell_unpack_kv(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, WT_CELL
 }
 
 /*
- * __wt_cell_unpack_delta_int --
+ * __wt_cell_unpack_delta_int_key --
  *     Unpack an internal delta cell into a structure.
  */
 static WT_INLINE void
-__wt_cell_unpack_delta_int(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *page_dsk,
-  const WT_PAGE_HEADER *dsk, WT_DELTA_CELL_INT *cell, WT_CELL_UNPACK_DELTA_INT *unpack_delta)
+__wt_cell_unpack_delta_int_key(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *page_dsk,
+  const WT_PAGE_HEADER *dsk, WT_CELL *cell, WT_CELL_UNPACK_DELTA_INT *unpack_delta)
 {
-    WT_DECL_RET;
-    const uint8_t *p;
-
     WT_UNUSED(dsk);
 
-    unpack_delta->flags = cell->__chunk[0];
-    p = (uint8_t *)&cell->__chunk[1];
-
     /* Unpack the key. */
-    __wt_cell_unpack_kv(session, page_dsk, (WT_CELL *)p, &unpack_delta->key);
-    p += unpack_delta->key.__len;
-
-    /* Optionally unpack the value if it exists. */
-    if (!F_ISSET(unpack_delta, WT_DELTA_INT_IS_DELETE)) {
-        __wt_cell_unpack_addr(session, page_dsk, (WT_CELL *)p, &unpack_delta->value);
-        p += unpack_delta->value.__len;
-    }
+    __wt_cell_unpack_kv(session, page_dsk, (WT_CELL *)cell, &unpack_delta->key);
+}
 
-    unpack_delta->__len = (uint32_t)WT_PTRDIFF(p, &cell->__chunk[0]);
+/*
+ * __wt_cell_unpack_delta_int_value --
+ *     Unpack an internal delta cell into a structure.
+ */
+static WT_INLINE void
+__wt_cell_unpack_delta_int_value(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *page_dsk,
```

Comment:
same here

Reply:
Fixed in 4841299c299ec85566e853eb4f7db12cb85e6408